packages/valory/skills/check_stop_trading_abci/rounds.py

```diff
@@ -73,6 +77,20 @@ class CheckStopTradingRound(VotingRound):
     no_majority_event = Event.NO_MAJORITY
     collection_key = get_name(SynchronizedData.participant_to_votes)
 
+    def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Enum]]:
+        """Process the end of the block."""
+        res = super().end_block()
+
+        if res is None:
+            return None
+
+        sync_data, event = res
+
```

Comment:
```suggestion

```

Reply:
I don't understand why this should be removed?

Reply:
Is it used in any way?